src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -683,6 +694,36 @@ ucs_status_t uct_rc_mlx5_ep_flush(uct_ep_h tl_ep)
     return UCS_INPROGRESS;
 }
 
+ucs_status_t uct_rc_mlx5_ep_fc_ctrl(uct_rc_ep_t *rc_ep)
```

Comment:
Sure, will change that. Used separate func, because FC checks were in uct_rc_mlx5_ep_inline_post in the first draft..